Library/Services/Implementation/NfieldSurveyGroupAssignmentsService.cs

```diff
@@ -0,0 +1,119 @@
+﻿//    This file is part of Nfield.SDK.
+//
+//    Nfield.SDK is free software: you can redistribute it and/or modify
+//    it under the terms of the GNU Lesser General Public License as published by
+//    the Free Software Foundation, either version 3 of the License, or
+//    (at your option) any later version.
+//
+//    Nfield.SDK is distributed in the hope that it will be useful,
+//    but WITHOUT ANY WARRANTY; without even the implied warranty of
+//    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//    GNU Lesser General Public License for more details.
+//
+//    You should have received a copy of the GNU Lesser General Public License
+//    along with Nfield.SDK.  If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Nfield.Infrastructure;
+using Nfield.Models;
+
+namespace Nfield.Services.Implementation
+{
+    internal class NfieldSurveyGroupAssignmentsService : INfieldSurveyGroupAssignmentsService, INfieldConnectionClientObject
+    {
+        public INfieldConnectionClient ConnectionClient { get; internal set; }
```

Comment:
Do you actually need to make this member internal if you have a public method to set it? I think you can make it private and just use the _Initialize_ to set it.

Reply:
Maybe you are right but all other existing service classes are internal as well, so I don't want to deviate from that.